.github/workflows/_build_publish.yaml

```diff
@@ -60,8 +64,20 @@ jobs:
           key: ${{ runner.os }}-${{ runner.arch }}-devtools-${{ hashFiles('mk/dependencies/deps.lock') }}
           restore-keys: |
             ${{ runner.os }}-${{ runner.arch }}-devtools
+      - name: Initialize CodeQL
```

Comment:
Shouldn't it be in check instead?

Reply:
The most time takes the build step which here is already done. If we'll put it in `check` we'll have to build binaries there as well which will take 10+ more minutes